analyzer/src/main/kotlin/PackageManager.kt

```diff
@@ -249,7 +249,7 @@ abstract class PackageManager(
             }
 
             log.info {
-                "Resolving $managerName dependencies for '${definitionFile.name}' took ${elapsed / 1000}s."
+                "Resolving $managerName dependencies for '${definitionFile.name}' took ${duration.inSeconds}s."
```

Comment:
Similar to the previous commit, maybe:
`"Resolved $manageName dependencies for '${definitionFile.name}' (took ${duration.inSeconds}s)."`

Reply:
Actually, I believe the emphasis of this log statement *should* be on the duration, so I'd prefer to not change the wording but only the implementation here.